openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2052,6 +2052,83 @@ regulation), the Credential Issuer should properly authenticate the Wallet and e
   </front>
 </reference>
 
+<reference anchor="W3C.Digital_Credentials_API" target="https://wicg.github.io/digital-credentials/">
+        <front>
+          <title>Digital Credentials API</title>
+		  <author fullname="Marcos Caceres">
+            <organization>Apple Inc.</organization>
+          </author>
+          <author fullname="Tim Cappalli">
+            <organization>Okta</organization>
+          </author>
+          <author fullname="Sam Goto">
+            <organization>Google</organization>
+          </author>
+        </front>
+</reference>
+
+# OpenID4VCI over the Digital Credentials API {#dc_api}
+
+This section defines how to use OpenID4VCI with the Digital Credentials API.
+
+The name "Digital Credentials API" (DC API) encompasses the W3C Digital Credentials API [@!W3C.Digital_Credentials_API]
+as well as its native App Platform equivalents in operating systems (such as [Credential Manager on Android](https://developer.android.com/jetpack/androidx/releases/credentials)).
+The DC API allows web sites and native apps acting as Issuers to initiate the issuance of verifiable credentials.
+The API itself is agnostic to the Credential issuance protocol and can be used with different protocols.
+The Web Platform, working in conjunction with other layers, such as the app platform/operating system, and based on the permission of the End-User, will send the credential offer along with the Origin of the Issuer to the End-User's chosen Wallet.
+
+## Protocol
+
+To use OpenID4VCI with the Digital Credentials API (DC API), the exchange protocol value has the following format: `openid4vci-v<version>`.
+
+The value `1` MUST be used for the `<version>` field to indicate the request and response are compatible with this version of the specification.
+
+The following exchange protocol values are defined by this specification:
+
+* `openid4vci-v1`
+
+## Credential Offer {#dc_api_credential_offer}
```

Comment:
I see potential for other requests, i.e. calling DC-API to continue a deferred issuance with `transaction_id`

Reply:
Do you mean like using the DC API to notify the wallet that a deferred issuance is now ready for issuance?

Reply:
That was my understanding too @tlodderstedt but I think @paulbastian is saying perhaps there is potential to use the DC API to notify a wallet about the change in status to a deferred credential issuance flow, like the credential is now ready to be picked up, did I get that right @paulbastian? If so I think there is quite a bit of complexity to unpack there before we could support it.